src/components/OsdComponent.vue

```diff
@@ -182,18 +182,31 @@ export default {
       // The users has selected an existing annotation
       // console.log('selected annotation')
       // console.log(annotation)
-      console.log("anno is clicked")  
+      console.log("anno is clicked " , annotation)  
```

Comment:
are the changes to this file relevant?

Reply:
2e41245472649c4ae228590604a58ce5ae196b08

Reply:
more or less the same as above: the changes here are irrelevant(?) for the PR, so please remove them.
In 2e41245 you only removed one change but not the all